dmsite/frontend/src/home/ClassificationColumn/ClassificationColumn.jsx

```diff
@@ -11,14 +11,24 @@ class ClassificationColumn extends Component {
     e.preventDefault();
   }
 
+  // TODO: at some point, base this off of the classification lookup, not on the file's classifications
+  isSensitive(val) {
+    if (val === true) { // == works, but === doesn't, for bools or numbers - need to fix
```

Comment:
=== is a strict comparison where as == will cast the values to be the same type before making the comparison. if === doesn't work val might not be necessarily be a boolean. Also, does `if (val) {...} not work?`

Reply:
`===` might not work here cause you're comparing 1 with true. `if (val)` should be sufficient because 0 is considered a falsy value and all other numbers are considered truthy.

Reply:
I did compare it with 1 and it still didn't work; this is just leftover from the last push. Don't quite know what to do honestly.